turtle_graphics/drawing_turtle_graphics-Style.js

```diff
@@ -0,0 +1,205 @@
+class Turtle {
+    constructor(x, y) {
+        this.presentPosition = [x, y]; // the present position
+        this.allPointsArr = []; // all the points in this array
+        this.allPointsArr.push(this.presentPosition);
+        this.sequentialPoints = []; // use for print()
+        let temp = [x, y];
+        temp.push(false);
+        this.sequentialPoints.push(temp);
```

Comment:
Here you are initializing `this.allPointsArr` and `this.sequentialPoints`, so you can set their initial values rather than initializing them as an empty array and pushing values to them in separate steps:
```
this.allPointsArr = [this.presentPosition];
this.sequentialPoints = [[x, y, false]];
```